plugins/veeValidate.ts

```diff
@@ -0,0 +1,21 @@
+import { Plugin } from '@nuxt/types/app';
+import { configure, extend } from 'vee-validate';
+import { required } from 'vee-validate/dist/rules';
+
+// Rules
+extend('required', required);
+
+extend('mustBeUrl', (value: string): boolean => {
```

Comment:
Small question here. Any reason to use this instead of the already provided [url validator](https://vee-validate.logaretm.com/v2/guide/rules.html#url) ?

Reply:
The rule provided by vee-validate seems to be base on [this](https://github.com/validatorjs/validator.js), and has a few options to have it handle URLs properly while not having to handle weird cases ourselves in the future.

Reply:
I'll switch - Since you were using a regex I wanted to have the exact same behavior and I forgot to investigate further.

Reply:
Looks like this rule is not available in v3: https://vee-validate.logaretm.com/v3/guide/rules.html#rules

Reply:
Indeed, for some reason, the docs I had was V2, not V3. We can look into moving the rule to something better at some point down the line, no big deal :)

Reply:
Do you mean a better regex check or something more flexible?

I can put:
```js
return /^https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_\+.~#?&//=]*)/.test(
    value
  );
```
```
http://something -> invalid
http://127.0.0.1 -> valid
https://server.local ->valid
```

Reply:
What about using `isUrl` from https://github.com/validatorjs/validator.js